utils/testtools/crypto.go

```diff
@@ -135,72 +78,72 @@ func CreateCSR(key crypto.PrivateKey) (csr []byte, err error) {
 	return csr, nil
 }
 
-// GetCACertificate returns CA certificate.
-func GetCACertificate() (cert []byte) {
-	return []byte(caCertificate)
-}
+// GenerateCertificate generate certificate.
+func GenerateCertificate(
+	template, parent *x509.Certificate, privateKey *rsa.PrivateKey, publicKey crypto.PublicKey,
+) ([]byte, error) {
```

Comment:
IMHO, returning *x509.Certificate is better idea because for example in fcrypt test, more often *x509.Certificate is required.

Reply:
Done